### PR TITLE
fix: org chart to TS

### PR DESCRIPTION
**Pull request by Oyunaa**
Updated org charts to TypeScript

### CODE REVIEW

File: frontend/src/components/orgchart/ExpandButton.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { CSSProperties } from "react";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import { IEmployee } from "../../interfaces/IEmployee";
+
+const expandBtn: CSSProperties = {
```
Comment (Narantsatsralt05) [inline frontend/src/components/orgchart/ExpandButton.tsx:6]: tus tusd n bish 
```js
const style = {
// backgroundColor: white
}
```

ingej hiivel busadtaigaa neg standard tai shuu

File: frontend/src/components/orgchart/ExpandButton.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  fontSize: "1rem",
+};
+
+const ExpandButton = (node: { data: { _directSubordinates: number }; children: IEmployee }) => {
+  console.log("expandButton", node.data._directSubordinates);
+
+  return (
+    <>
+      {node && (
+        <div style={expandBtn}>
```
Comment (Narantsatsralt05) [inline frontend/src/components/orgchart/ExpandButton.tsx:31]: Ali boloh MUI component uudaa ashiglaarai

File: frontend/src/components/orgchart/NodeContent.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            </div>
+          </div>
+        ) : (
+          <div style={nodeTeam}>
+            <div style={nodeTeamName}>{props.data.team}</div>
+            {props._children !== null &&
+              props._children
+                .slice(0, 4)
+                .map(child => (
+                  <img key={child.data.id} style={nodeTeamMemberImg} src={child.data.imageUrl} alt="team member" />
```
Comment (Narantsatsralt05) [inline frontend/src/components/orgchart/NodeContent.tsx:99]: Next/image iig ashiglasan n davuu taltai baidag shuu doc oos n unshaarai 😃

File: frontend/src/components/orgchart/OrganizationalChart.tsx [hunk 4]
```diff
@@ -34,9 +35,11 @@ const OrganizationalChart = props => {
           toggleDetailsCard(d);
         })
         .buttonContent(node => {
+          console.log(node);
```
Comment (Narantsatsralt05) [inline frontend/src/components/orgchart/OrganizationalChart.tsx:38]: Remove console.log

File: frontend/src/pages/orgchart/index.tsx [hunk 5]
```diff
@@ -1,8 +1,20 @@
...
+import React, { useEffect, useState } from "react";
+import data from "./data";
 import OrganizationalChart from "../../components/orgchart/OrganizationalChart";
+import { IEmployee } from "../../interfaces/IEmployee";
 
 const OrgChart: NextPage = () => {
+  const [employees, setEmployees] = useState<IEmployee[]>([]);
+
+  useEffect(() => {
+    console.log(data);
```
Comment (Narantsatsralt05) [inline frontend/src/pages/orgchart/index.tsx:11]: Remove console logs

**Comment by Narantsatsralt05** [comment]
Nice PR project status aa oorchlood, zurag oruulchih ym bol bur nice boloh n

**Comment by billduluu** [comment]
@enkhbayar-gan eniig hze awh bolomjtoi bga ve

File: frontend/src/components/orgchart/NodeContent.tsx (hunk 3 above)
Comment (enkhbayar-gan) [reply frontend/src/components/orgchart/NodeContent.tsx]: next image hetsu bh dara boloy

Comment (enkhbayar-gan) [reply frontend/src/components/orgchart/NodeContent.tsx]: i mean there's a problem with build or file config issue. I don't remember exactly.

**Pull request merged**